libs/ui/src/centered_large_prose.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Prose } from './prose';
+import { fontSizeTheme } from './themes';
+
+interface CenteredLargeProseProps {
+  children: React.ReactNode;
+}
+
+export function CenteredLargeProse({
+  children,
+}: CenteredLargeProseProps): JSX.Element {
+  return (
+    <Prose textCenter maxWidth={false} themeDeprecated={fontSizeTheme.large}>
+      {children}
+    </Prose>
```

Comment:
Just to make sure I understand correctly, is usage of `themeDeprecated` in `libs/ui` less compatible/more strict than usage in `scan/frontend`?

Reply:
Nope, I know you're just porting this over. But @kofi-q's been working on `scan/frontend` in an NH-specific branch and I would guess that he touched this theming. I'm just calling out a merge conflict that he or whoever merges that branch will have to deal with, because it may require some special attention.

Reply:
Understood, thanks! @kofi-q let me know if you have any issues, happy to discuss these changes and work through any merge conflicts